spec/models/meeting_option_spec.rb

```diff
@@ -40,15 +40,15 @@
           create(:rooms_configuration, meeting_option:, provider: 'greenlight', value: 'optional')
 
           rooms_config = described_class.get_config_value(name: 'setting', provider: 'greenlight')
-          expect(rooms_config.value).to eq('optional')
+          expect(rooms_config['setting']).to eq('optional')
```

Comment:
Um, nice!
We may even test this multi configs fetching capability?

Reply:
Isn't it being tested already since we are returning multi configs in `create_meeting_options` and a single config in `room_settings#update` ?

Reply:
Are we testing that in the `create_meeting_options` tests?
However, that's an integration test I think we may like to initially target that in its unit test catching all cases,
In example, in the `create_meeting_options` test you were only creating a `glViewerAccessCode` meeting option record only so to ensure that the multi configs fetching is working is only because we know the implementation of the actual method,